analyze.c

```diff
@@ -6,23 +6,53 @@
  */ 
 #include "analyze.h"
 
-float pitch_est(AxesSI_t Axes){
-	float pitch;
+int32_t get_pitch(AxesSI_t* Axes){
+	int32_t pitch = 0;
 	/*Apply trigonometry to get the pic*/
-	pitch = (atan(Axes.yAxis/sqrt(pow(Axes.zAxis,2)+pow(Axes.xAxis,2))))*(180.0/PI);
+	pitch = (atan(-Axes->yAxis/sqrt(pow(Axes->zAxis,2)+pow(Axes->xAxis,2))))*(180.0/PI);
 	return pitch;
 }
 
-float roll_est(AxesSI_t Axes){
-	float roll;
+int32_t get_roll(AxesSI_t* Axes){
+	int32_t roll = 0;
 	/*Apply trigonometry to get the pic*/
-	roll = (atan(-Axes.xAxis/Axes.zAxis))*(180.0/PI);
+	roll = (atan(-Axes->xAxis/Axes->zAxis))*(180.0/PI);
 	return roll;
 }
-/*
-float yaw_est(AxesSI_t Axes){
-	float yaw;
-	yaw = (atan(Axes.zAxis/sqrt(pow(Axes.xAxis,2)+pow(Axes.yAxis,2))))*(180.0/PI);
+
+int32_t get_yaw(AxesSI_t* Axes){
+	int32_t yaw = 0;
+	yaw = (atan(Axes->zAxis/sqrt(pow(Axes->xAxis,2)+pow(Axes->yAxis,2))))*(180.0/PI);
 	return yaw;
 }
-*/
\ No newline at end of file
+/*
```

Comment:
this code needs to be either deleted or used. there should be no unreachable code, and commented out code is a big red. refer to the style guide.